op-batcher/batcher/espresso.go

```diff
@@ -184,6 +184,10 @@ func (l *BatchSubmitter) espressoBatchLoadingLoop(ctx context.Context, wg *sync.
 					continue
 				}
 
+				for _, txn := range block.Transactions() {
```

Comment:
We should definitely be careful about leaving these logs in the final implementation. If we do think that they are helpful for debugging, then we should, at the very least, demote their log levels.

Reply:
Yes, these log messages have been removed in e2ecdcf3b.